desk/app/groups.hoon

```diff
@@ -3070,6 +3070,15 @@
     =?  net  ?=(%sub -.net)
       [%sub time.net &]
     =?  go-core  !was-init
+      ::  initialize active-channels on group init
+      ::
+      =/  nests
+        ~(tap in ~(key by channels.group))
+      =?  active-channels.group  !=(~ nests)
+        %-  silt
+        %+  skim  nests
+        |=  =nest:g
+        .^(? %gu (channels-scry nest))
```

Comment:
This does make us more dependent on correctly removing channels from channels agent state when we leave a channel, which kind of invalidates the point of the `active-channels` set, no?

@arthyn can I get a second opinion from you on the behavior/implications here?

Reply:
I don't think channels being the de facto source of truth necessarily invalidates active-channels. The point was to have the set of joined channels tracked in the group model for the client. This is more convenient especially with the current changes endpoints and the data they return.

Regardless yes we'll have to make sure channels leaves when appropriate